spec/factories/lists/establishment.rb

```diff
@@ -3,6 +3,14 @@
     establishment_name { 'sample' }
     postcode { 'AB12 34C' }
     number_of_pupils { 100 }
-    last_changed_date { 1.month.ago }
+    open_date { DateTime.current }
+  end
+
+  factory :closed_establishment, class: 'Lists::Establishment' do
```

Comment:
If you nest the `:closed_establishment` factory within `:establishment` then FactoryBot uses the outer factory to define all the main fields, then you can customise to add a close date, which saves some repetition. See users_factory and look at `:user` and `:pupil`.